test/e2e/internal/utils/exec.go

```diff
@@ -121,6 +121,12 @@ func (opts *ExecOption) MatchErrKeyWords(keywords ...string) *ExecOption {
 	return opts
 }
 
+// MatchRequestKeywords adds keywords matching to each sent request.
+func (opts *ExecOption) MatchRequestKeywords(keywords ...string) *ExecOption {
```

Comment:
We need more like a `func (opts *ExecOption) MatchRequestHeader(headers map[string]string)`

Reply:
Renamed this function to `MatchRequestHeaderKeywords`.